sql/parse/parse.go

```diff
@@ -948,39 +948,35 @@ func isAggregateFunc(v *sqlparser.FuncExpr) bool {
 // base, to its smallest representation possible, out of:
 // int8, uint8, int16, uint16, int32, uint32, int64 and uint64
 func convertInt(value string, base int) (sql.Expression, error) {
-	i8, err := strconv.ParseInt(value, base, 8)
-	if err != nil {
-		ui8, err := strconv.ParseUint(value, base, 8)
-		if err != nil {
-			i16, err := strconv.ParseInt(value, base, 16)
-			if err != nil {
-				ui16, err := strconv.ParseUint(value, base, 16)
-				if err != nil {
-					i32, err := strconv.ParseInt(value, base, 32)
-					if err != nil {
-						ui32, err := strconv.ParseUint(value, base, 32)
-						if err != nil {
-							i64, err := strconv.ParseInt(value, base, 64)
-							if err != nil {
-								ui64, err := strconv.ParseUint(value, base, 64)
-								if err != nil {
-									return nil, err
-								}
-								return expression.NewLiteral(uint64(ui64), sql.Uint64), nil
-							}
-							return expression.NewLiteral(int64(i64), sql.Int64), nil
-						}
-						return expression.NewLiteral(uint32(ui32), sql.Uint32), nil
-					}
-					return expression.NewLiteral(int32(i32), sql.Int32), nil
-				}
-				return expression.NewLiteral(uint16(ui16), sql.Uint16), nil
-			}
-			return expression.NewLiteral(int16(i16), sql.Int16), nil
-		}
-		return expression.NewLiteral(uint8(ui8), sql.Uint16), nil
+	if i8, err := strconv.ParseInt(value, base, 8); err == nil {
+    return expression.NewLiteral(int8(i8), sql.Int8), nil
```

Comment:
i think your editor did not run gofmt to this part

Reply:
:thinking: It looks good on my side, I'll check

Reply:
Spaces instead of tabs, ofc. Force pushed as https://github.com/src-d/go-mysql-server/pull/840/commits/3034d2d03a9d2a485470e630873fa40b9d87f61e